docs/docsite/rst/conf.py

```diff
@@ -28,6 +28,11 @@
 sys.path.insert(0, os.path.join('ansible', 'lib'))
 sys.path.append(os.path.abspath(os.path.join('..', '_extensions')))
 
+# We want sphinx to document the ansible modules contained in this repository,
+# not those may happen to be installed in the version of Python used to run sphinx.
+# As sphinx loads in order to document, the repository version needs to be loaded:
+sys.path.insert(0, os.path.abspath(os.path.join('..', '..', '..', 'lib')))
+
 VERSION = '2.7'
```

Comment:
Interesting, would expect this to be 2.8 on `devel`, perhaps it isn't used by anything.

Reply:
@gundalow I'm not sure what that does - I updated it (from 2.6) as part of the 2.7 release, thinking perhaps the `VERSION` setting in the Sphinx config should match the `latest` setting on the docsite. We could experiment . . .

Reply:
Being curious, I did an experiment, namenly, setting this VERSION to `3.14`, then running `make clean && make webdocs` and `grep`ing all files for `3.14`. Lessons learned:

"3.14" was not a good choice, as many tests contain an IP 11.12.13.14.

Every single .html - file generated contains a piece of javascript like this:

```
    <script type="text/javascript">
      var DOCUMENTATION_OPTIONS = {
        URL_ROOT:'./',
        VERSION:'3.14',
        COLLAPSE_INDEX:false,
        FILE_SUFFIX:'.html',
        HAS_SOURCE:  false
      };
    </script>
```
, checked by
```
$ for f in $(find docs/docsite/_build/html -name \*.html); do grep -q "VERSION:'3.14'" $f || echo $f; done
$
```

Reply:
Thanks for running the experiment, @aknrdureegaesr. I'll open a separate PR to update the version to 2.8 on `devel`.